Packager/CMakeLists.txt

```diff
@@ -18,8 +18,9 @@
 set(PLUGIN_NAME Packager)
 set(MODULE_NAME WPEFramework${PLUGIN_NAME})
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
 find_package(${NAMESPACE}Plugins REQUIRED)
-find_package(libprovision REQUIRED)
```

Comment:
Can we please make this conditional. We have customers using this packager relying on libprovision in combination with OPKG. So suggest to make this package optional, and if found, build it the old way, if not do it the comcast used way.
This way you do not break current deplyments